File: src/main/golang/leetcode/editor/cn/[172]Factorial Trailing Zeroes.go
package lc

//leetcode submit region begin(Prohibit modification and deletion)
func trailingZeroes(n int) (ans int) {
	for n > 0 {
		n /= 5
		ans += n
	}
	return
}

//leetcode submit region end(Prohibit modification and deletion)

File: src/main/golang/leetcode/editor/cn/[172]Factorial Trailing Zeroes.py
# leetcode submit region begin(Prohibit modification and deletion)
class Solution:
    def trailingZeroes(self, n: int) -> int:
        ans = 0
        while n > 0:
            n //= 5
            ans += n
        return ans
# leetcode submit region end(Prohibit modification and deletion)
